spec/abide_dev_utils/xccdf_spec.rb

```diff
@@ -20,7 +20,7 @@
     ctrl_num_fmt: 'c1_1_1_3'
   },
   'Windows' => {
-    benchmark: nil,
+    benchmark: '',
```

Comment:
why did this need to change?

Reply:
It didn't, leftovers from me testing stuff